packages/fetchai/protocols/fipa/dialogues.py

```diff
@@ -60,32 +50,43 @@
             FipaMessage.Performative.DECLINE,
         ]
     ),
+    FipaMessage.Performative.CFP: frozenset(
+        [FipaMessage.Performative.PROPOSE, FipaMessage.Performative.DECLINE]
+    ),
+    FipaMessage.Performative.DECLINE: frozenset(),
+    FipaMessage.Performative.INFORM: frozenset([FipaMessage.Performative.INFORM]),
     FipaMessage.Performative.MATCH_ACCEPT: frozenset([FipaMessage.Performative.INFORM]),
     FipaMessage.Performative.MATCH_ACCEPT_W_INFORM: frozenset(
         [FipaMessage.Performative.INFORM]
     ),
-    FipaMessage.Performative.INFORM: frozenset([FipaMessage.Performative.INFORM]),
-    FipaMessage.Performative.DECLINE: frozenset(),
+    FipaMessage.Performative.PROPOSE: frozenset(
+        [
+            FipaMessage.Performative.PROPOSE,
+            FipaMessage.Performative.ACCEPT,
+            FipaMessage.Performative.ACCEPT_W_INFORM,
+            FipaMessage.Performative.DECLINE,
+        ]
+    ),
 }  # type: Dict[FipaMessage.Performative, FrozenSet[FipaMessage.Performative]]
```

Comment:
The order is different because originally there was no order but the generator now uses sorting to ensure the order is fixed across repeated generations.